recipe/run_test.sh

```diff
@@ -18,6 +20,6 @@ if [[ $target_platform == "linux-64" ]]; then
 
     git clone https://github.com/NVIDIA/CUDALibrarySamples.git sample_linux/
     cd sample_linux/cuSPARSELt/matmul
-    error_log=$(nvcc --std=c++14 -I$CUDA_INC_PATH -I$PREFIX/include -L$PREFIX/lib -lcusparse -lcusparseLt -lnvrtc matmul_example.cpp -o matmul_example 2>&1)
+    error_log=$(nvcc --std=c++14 -I$CUDA_INC_PATH -I$PREFIX/include -L$PREFIX/lib -L$PREFIX/lib/stubs -lcusparse -lcusparseLt -lcuda -lnvrtc matmul_example.cpp -o matmul_example 2>&1)
```

Comment:
Adding the CUDA driver directory to link directories.

Note: consider 
- → expanding this test to include more samples
- → separating and isolating common compilation logic
- → defer compilation logic to `cmake` or similar build system native to CUDA library samples.

Reply:
cc @kvoronin